python/tests/test_highlevel.py

```diff
@@ -851,6 +851,52 @@ def test_minlex_postorder_multiple_roots(self):
         assert tree_orders == expected_result
 
 
+class TestMRCA:
+    t1 = tskit.Tree.generate_balanced(3)
+    #  4
+    # ┏━┻┓
+    # ┃  3
+    # ┃ ┏┻┓
+    # 0 1 2
+
+    t2 = tskit.Tree.generate_balanced(2)
+    # 2
+    # ┏┻┓
+    # 0 1
+
+    def test_two_or_more_args(self):
+        t2 = self.t2
```

Comment:
There's probably not much point in assinging to a local variable here, can just do
```
self.tree.mrca(0, 1) == 2
# etc
```